modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/chapters/ChaptersFragment.kt

```diff
@@ -82,6 +85,18 @@ class ChaptersFragment : BaseFragment() {
                         }
                 }
 
+                LaunchedEffect(Unit) {
+                    chaptersViewModel
+                        .snackbarMessage
+                        .collectLatest { message ->
+                            Toast.makeText(
+                                context,
+                                context.getString(message),
+                                Toast.LENGTH_SHORT,
+                            ).show()
```

Comment:
Shouldn't this be a snackbar to match the design?

Reply:
Yep, replaced toast with snackbar

<img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/a948ff71-6737-46e8-97af-3c0a01b91a48"/>